docs/core/man/doveconf.1.md

```diff
@@ -47,34 +47,6 @@ configuration in easy human readable output.
 :   Show the setting's default value instead of the one currently
     configured.
 
-**-f** *filter*
```

Comment:
-f parameter does work. I added it back a month or two ago.

Reply:
Commit dropped. :saluting_face: